apps/web/src/pages/Home/HomePage/components/Hero/Hero.tsx

```diff
@@ -89,28 +89,30 @@ const Hero = ({ defaultImage }: HeroProps) => {
           data-h2-justify-content="base(center) p-tablet(flex-start)"
           data-h2-flex-wrap="base(wrap) p-tablet(initial)"
         >
-          <CallToActionLink
+          <Link
             color="quaternary"
-            Icon={JobIcon}
+            mode="cta"
+            icon={MagnifyingGlassIcon}
```

Comment:
So, the icon names changed since they were custom and have been replaced by the heroicon equivalents. You should notice that they are the exact same icon, just imported from `@heroicons/react` with a different name.

I checked and confirmed this for all other icons you noted.